backend/src/errors.rs

```diff
@@ -24,12 +24,16 @@ impl fmt::Debug for ParserError {
 
 pub enum TangleError {
     BlockNotFound(String),
+    LanguageNotSupported,
```

Comment:
Let's make this contain the incorrect value
```suggestion
    LanguageNotSupported(String),
```